### PR TITLE
Adding postgres

### PR DESCRIPTION
**Pull request by HarshwardhanSingh**
Create root route to pages#welcome. 
Added and configured pg gem for development and production both environment to deploy on heroku.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,12 @@
...
-
-* ...
-# leave-tracker
+# Leave-tracker
+
+Leave tracker is a Rails 5 app built for tracking the leaves taken by the employees of the organization.
+
+## How to install
+1. Download the repository.
+2. Extrach the zip folder.
```
Comment (iffyuva) [inline README.md]: what do you mean by extracting zip?

File: config/database.yml [hunk 2]
```diff
@@ -1,26 +1,85 @@
...
+# You can use this database configuration with:
+#
+#   production:
+#     url: <%= ENV['DATABASE_URL'] %>
+#
 production:
   <<: *default
-  adapter: postgresql
-  database: db/production.sqlite3
+  database: testpg_production
```
Comment (revathskumar) [inline config/database.yml]: DB name should be leave_tracker

File: config/routes.rb [hunk 3]
```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+root to: 'pages#welcome'
```
Comment (revathskumar) [inline config/routes.rb]: Fix indentation.

File: Gemfile [hunk 4]
```diff
@@ -28,15 +28,15 @@ gem 'jbuilder', '~> 2.0'
...
 
-group :production do
-  gem 'pg'
-end
 
 group :development, :test do
   # Use sqlite3 as the database for Active Record
-  gem 'sqlite3'
+  # gem 'sqlite3'
+  gem 'rspec', '~> 3.4'
```
Comment (emilsoman) [inline Gemfile]: Please remove rspec from this PR. We'll set up rspec-rails in a different PR

File: README.md (hunk 1 above)
Comment (revathskumar) [inline README.md]: @HarshwardhanSingh We setup using `git clone`. see the readme of https://github.com/code-mancers/integration-diff

File: Gemfile [hunk 5]
```diff
@@ -28,15 +28,14 @@ gem 'jbuilder', '~> 2.0'
...
 gem 'rails_12factor'
 
-group :production do
-  gem 'pg'
-end
 
 group :development, :test do
   # Use sqlite3 as the database for Active Record
-  gem 'sqlite3'
+  # gem 'sqlite3'
```
Comment (revathskumar) [inline Gemfile]: @HarshwardhanSingh There is no reason to keep this as a comment. Please remove it.

Comment (HarshwardhanSingh) [inline Gemfile]: @revathskumar done.
Please check.

**Comment by revathskumar** [comment]
The travis is failing because no `.travis.yml`.
PR LGTM.
Hence merging.

**Pull request merged**